src/backends/fixerio/index.js

```diff
@@ -32,9 +31,6 @@ class FixerIoBackend {
     this.rates = {}
     this.currencies = []
     const ledgerPairs = config.get('tradingPairs')
-    if (_.isEmpty(ledgerPairs)) {
```

Comment:
This was a "safety check" that Chris and I decided to add. Live and learn, I guess ...